src/homework/client/src/components/Main.js

```diff
@@ -136,14 +83,29 @@ function Main() {
 					onChange={e => setTask(e.target.value)}
 					onKeyPress={handleKeyPress}
 				/>
+				
 				<button className="btn" id="add-btn" onClick={handleAddTodo}>
 					+
 				</button>
+			
 			</div>
 
-			<br />
-			<List data={list} reload={reload} />
+			<div>
+			<input
+					type= "text"
+					id="input-search-task"
+					placeholder="Search..."
+					style={{ marginLeft: 70 }}
+					value={search}
+					onChange={e => setSearch(e.target.value)}
+				/>
+				<button className="btn" id="search-btn" onClick={handleAddTodo}>
+					+
+				</button>
+			</div>
 
+			<br />
+			<ListConnector search={search}/>
```

Comment:
I would rename search to "searchTerm".
Search is sound like an action/method while it's a string. confusing.